src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java

```diff
@@ -350,4 +335,51 @@ void GIVEN_log_manager_in_errored_state_WHEN_restarted_THEN_logs_upload_is_reatt
         }
         assertEquals(1, allFiles.size());
     }
-}
+
+    @Test
```

Comment:
https://www.baeldung.com/junit-5-repeated-test

Let's repeat this several times to make sure that we are doing the right thing, as much as we can.